docs/index.asciidoc

```diff
@@ -0,0 +1,221 @@
+[[plugins-filters-dissect]]
```

Comment:
For links within a single book, asciidoc only cares about the IDs, so I _think_ you would be OK.